docs/userguide/targets.md

```diff
@@ -1,18 +1,87 @@
 # Targets
 
-## General information
+Targets play a crucial role as the connection between observables measured in an
+experiment and the machine learning core behind BayBE.
+In general, it is expected that you create one [`Target`](baybe.targets.base.Target)
+object for each of your observables.
+The way BayBE treats multiple targets is then controlled via the 
+[`Objective`](../../userguide/objective).
 
-For BaybBE, targets play a crucial role in modeling the function that requires optimization during the DoE campaign.
-Targets are used to model the function that should be optimized by the DoE campaign. They are implemented via the [`Target`](baybe.targets.base.Target) class.
-Each target has a name as well as an abstract transform function. This function is used to transform data into computational representation. 
+## NumericalTarget
+Besides the `name`, a [`NumericalTarget`](baybe.targets.numerical.NumericalTarget)
+has the following attributes:
+* **The optimization** `mode`: This specifies whether we want to minimize or maximize
+  the target or whether we want to match a specific value.
+* **Bounds:** Define `bounds` that constrain the range of target values.
+* **A transformation function:** When bounds are provided, a `transformation` is
+  used to map target values onto the [0,1] interval.
 
-BayBE currently supports a single type of target, called [`NumericalTarget`](baybe.targets.numerical.NumericalTarget), specifically designed for numerical modeling.
+Below a visualization of possible choices for `transformation`, where `lower` and
+`upper` are the entries provided via `bounds`:
+![Transforms](../_static/target_transforms.svg)
 
-## Numerical targets
+### MIN and MAX mode
+Here two examples for simple maximization and minimization targets:
```

Comment:
Make this a proper sentence.

Reply:
changed ( I still do consider the the old sentence proper, it is called telegraphic style)